server/chat-plugins/helptickets.ts

```diff
@@ -208,7 +208,7 @@ export class HelpTicket extends Rooms.RoomGame {
 
 	modnote(user: User, text: string) {
 		this.room.addByUser(user, text).update();
```

Comment:
Wait, is this supposed to be public?

Reply:
I thought modnotes were mod-only.

Reply:
Perhaps it's different in help tickets, since only the global staff handling the ticket and the person making the ticket would be in the room?

Reply:
Modnotes in regular rooms are definitely mod-only.

Reply:
This function should probably not be called "modnote", so it's not confused with the mod-only modnotes.

Reply:
But that can be a separate PR>